utils/entitlements.go

```diff
@@ -75,7 +74,7 @@ func ProjectEntitlementsByBundleID(pth, schemeName, configurationName string) (m
 		return nil, fmt.Errorf("target not found: %s", archiveEntry.BuildableReference.BlueprintIdentifier)
 	}
 
-	targets := append([]xcodeproj.Target{mainTarget}, mainTarget.DependentExecutableProductTargets()...)
+	targets := append([]xcodeproj.Target{mainTarget}, mainTarget.DependentExecutableProductTargets(false)...)
```

Comment:
Can you elaborate on why was the `false` parameter introduced?

Reply:
This was the given method before the dep update:
https://github.com/bitrise-steplib/steps-xcode-archive/blob/master/vendor/github.com/bitrise-io/xcode-project/xcodeproj/target.go#L50
and this after the dep update:
https://github.com/bitrise-steplib/steps-xcode-archive/blob/TOOL-20/vendor/github.com/bitrise-io/xcode-project/xcodeproj/target.go#L51
Basically, the if condition was extended with an "AND". To maintain the former behaviour, decided to set the bool to false, as with that the second part will be always true.